package.json

```
@@ -54,6 +54,11 @@
 					"type": "string",
 					"description": "The contracts source directory within your project, for example: src"
 				},
+				"cairols.disableHintValidation": {
+					"scope": "resource",
+					"type": "string",
+					"description": "Disables hint validation for selected files"
```

This should describe the expected format of this setting, and it would also be useful to provide an example.

```suggestion
					"description": "Disables hint validation for files that match the given pattern, for example: tests/**/*.cairo"
```